packages/web3-shared/solana/src/utils/address.ts

```diff
@@ -43,7 +43,7 @@ export function isValidAddress(address?: string) {
     const length = address?.length
     if (!length) return false
     try {
-        return length >= 32 && length <= 44 && Web3.PublicKey.isOnCurve(bs58.decode(address))
```

Comment:
Strangely, even the phantom ext cannot accept your given address.

<img width="410" alt="image" src="https://user-images.githubusercontent.com/52657989/196211944-ba8096e3-ae2b-41fd-8f53-c3bbac10719f.png">

I did some research about it. Found such address could be more:

+ `4ig5eTKvBefXB9Pqwq2rCJTGFRNATEa7geNzeuVQGmAE`
+ `Eej78Z5E4w5sSreb8w8esQqTVwBuyT8La2ypa8jiY3Tw`

A Solana explorer page could be found. However, the `isOnCurve` method returns `false`.